### ISSUE TITLE
SoftLayer machine update before Round 2

### ISSUE THREAD

**Issue by tschaffter** [Sage-Bionetworks/DigitalMammographyChallenge#434]
SoftLayer requested to update several packages on the machines. The update is now performed during the maintenance week between Round 1 and Round 2. The list below will be updated to reflect the progress of the update.

- [x]  bm01
- [x]  bm02
- [x]  bm03
- [x]  bm04
- [x]  bm05
- [x]  bm06 (docker 1.12.6, others have 1.12.5)
- [x]  bm07
- [x]  bm08
- [x]  bm09
- [x]  bm10
- [x]  bm11
- [x]  bm12
- [x]  bm13
- [x]  bm14
- [x]  bm15
- [x]  bm16
- [x]  bm17
- [x]  bm18
- [x]  bm19
- [x]  bm20
- [x]  bm21
- [x]  bm22
  - [x] disk replacement
  - [x] create LVs
  - [x] deploy dataset (Jan 2-3rd)
- [x]  bm23

Once a machine has been updated by SL, the following protocol is followed before restarting the Challenge agent: #387

**Comment by tschaffter** [comment]
The update is postponed for the following machines: bm06, bm13 (Darvin) and bm16.

**Comment by tschaffter** [comment]
@brucehoff All the machines that are not in used have been updated. Let's discuss before restarting the agents.

**Comment by tschaffter** [comment]
Remaining machines to update: bm06, bm13 (Darvin) and bm16. Darvin is running jobs but he says that the machine should be available to perform the update sometime today.

@brucehoff SoftLayer is asking me to provide a maintenance window for bm06 and bm16. Do you have an idea of when that could be?

**Comment by brucehoff** [comment]
> Do you have an idea of when that could be?
Each of bm06 and bm16 is running two submissions.  I looked at the logs for the two bm06 submissions but didn't see any indication of how far along they are. 

On bm16 I looked at the log file for 7877006.  It seems to be on subject 14202 and before that 14198.  Submission 7904505 has almost nothing in its log file.  It's hard to believe it's doing anything at all.

**Comment by tschaffter** [comment]
From Bruce: "bm06 is done running scoring submissions and can now be reclaimed for maintenance."

**Comment by tschaffter** [comment]
@brucehoff bm06 and bm16 still need to be updated. You sent an email saying that bm06 can be updated. I also check bm16 and no challenge agents are running on it. Can I update both and close this ticket? Thanks!

**Comment by brucehoff** [comment]
> Can I update both and close this ticket? 

@tschaffter YES.  Please let me know when I can restart the agents on the machines.

**Comment by tschaffter** [comment]
Finally, bm06 and bm16 have been updated by the DST. I've followed the protocol referenced above to cleanup these two predictions machines. When running `./DigitalMammographyChallenge/setupNewServer.sh softlayer bm06`, bm06 got a version of docker more recent (1.12.6) than the other machines (1.12.5). The sysdig and nvml-statsd containers were also stated using `./DigitalMammographyChallenge/setupSysdigAgent.sh softlayer bm0X`.

@brucehoff The maintenance of the SoftLayer machines is now completed. bm06 and bm16 can now be reconnected.
